smarty/templates/email/issue_change.tpl

```diff
@@ -0,0 +1,11 @@
+Subject: Change to Issue # - {$issueID}
+
+{$realname},
+
+There has been an update to an issue you are watching.
+
+Please view the changes <a href="{$url}">here</a>
```

Comment:
I'm not sure if the Email function allows HTML in email. This should probably just say "Please view the changes at {$url}"